docs/Transactions.md

```diff
@@ -14,24 +14,24 @@ You initialize a transaction by making an async call to `producer.transaction()`
 > Note: Kafka requires that the transactional producer have the following configuration to _guarantee_ EoS ("Exactly-once-semantics"):
 >
 > - The producer must have a max in flight requests of 1
-> - The producer must wait for acknowledgement from all replices (acks=-1)
-> - The producer must have unlimitted retries
+> - The producer must wait for acknowledgement from all replicas (acks=-1)
+> - The producer must have unlimited retries
 
-Configure the client with `maxFlightRequests: 1` to guarantee EOS.
+Configure the producer client with `maxFlightRequests: 1` to guarantee EOS.
```

Comment:
The producer needs to have both `maxInFlightRequests: 1` and `idempotent: true` to guarantee EOS, so I would recommend merging this sentence with the one below ("Setting `idempotent` to `true`...") and getting rid of this code block, so that you just have one sentence and one code block with everything you need to configure your producer for EOS.

Reply:
Also, it's `maxInFlightRequests`, not `maxFlightRequests`

Reply:
adjusted the docs, thanks.